lms/djangoapps/completion/models.py

```diff
@@ -217,3 +217,19 @@ def __unicode__(self):
             block_key=self.block_key,
             completion=self.completion,
         )
+
+
+class UserSalesforceContactId(models.Model):
+    """
+    Model to track the Salesforce Contact ID 18 user value.
+
+    The field contact id is unique for each user.
+
+    The user contact id was initially stored in the model UserSocialAuth extra_data.
+
+    A new Django command is available to grab the user contact id from UserSocialAuth
+    and then create the new record in this model.
+    """
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    contact_id = models.CharField(max_length=60)
```

Comment:
do you know how the contact id structure is ?, can you ensure that the contact-id won't be longer than 60 ?

Reply:
Maybe this could be done in the sing up process, and then specified in the ticket. @andrey-canon